design.md

```diff
@@ -2,12 +2,26 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We are targetting CS5 students with this design. We assume the user has a basic understanding of programming.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+We chose this design so that users can write more readable picobot code.
+This way, it is easier to interpret and debug!
+
```

Comment:
seems like it might be a more intuitive style for someone who already has a degree of comfort with automata and control flow, but maybe not for intro level programmers?